tests/unit/test_load.py

```diff
@@ -0,0 +1,19 @@
+import os
+from datetime import datetime
+
+from google.cloud.storage.bucket import Bucket
+
+
+def test_load_get_bucket(load):
+    assert type(load._get_bucket()) == Bucket
+    assert load._get_bucket().name == os.getenv("BUCKET")
+
+
+def test_load_create_blob(coinmarket_response, load, bucket):
+    blob_name = f"{datetime.today().strftime('%Y-%m-%d')}-unit-test.json"
+    load.create_blob(name=blob_name, crypto_data=coinmarket_response)
+    blob = bucket.blob(blob_name)
+    blob_exists = blob.exists()
+    assert blob_exists
```

Comment:
ah I see why the extra var, to not call `blob.exists()` twice right?

Reply:
yeah, it was kinda funky. Was late when I wrote this so not the greatest implementation and mainly wanted to get it working before signing off. I'm going to rework the test/approach.

Reply:
ok cool, let me know if you get stuck on anything ...